libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Identity/SFIdentityData.m

```diff
@@ -65,6 +65,7 @@
 NSString * const kSFIdentityCustomAttributesKey           = @"custom_attributes";
 NSString * const kSFIdentityCustomPermissionsKey          = @"custom_permissions";
 NSString * const kSFIdentityLastModifiedDateKey           = @"last_modified_date";
+NSString * const kSFNativeLoginDateKey                    = @"native_login";
```

Comment:
could this be renamed if it's not used to store a date?

Reply:
Yes!